src/main/java6/net/finmath/marketdata/model/curves/ForwardCurveInterface.java

```diff
@@ -34,12 +41,15 @@
     double getForward(AnalyticModelInterface model, double fixingTime, double paymentOffset);
 
     /**
-	 * Returns the name of the discount curve associated with this forward curve.
-	 * 
-	 * @return The name of the discount curve associated with this forward curve.
-	 */
+     * @return The discountCurveName if forwardCurve is ForwardCurve, null otherwise
```

Comment:
I do not understand the JavaDoc here. The intent of a disocuntCurveName field in a forward curve is the following: A forward (e.g. EURIBOR 3M) which is calibrated from collateralised products (e.g. EUR OIS) is ultimately associated with this discount curve. So for EURIBOR 3M the disocuntCurveName should give EUR OIS.
In an single curve environment such a curve may not be available and result NULL.
Note also that the concept of an associated discountCurve is motivated from the interpolation methods discussed in the "Curves" paper. This discountCurveName should stay. The JavaDoc change is misleading.